apps/drec-api/package.json

```diff
@@ -4,32 +4,50 @@
     "start:prod": "node dist/js/src/main",
     "prebuild": "shx rm -rf dist",
     "build": "yarn build:ts",
     "build:ts": "tsc -b tsconfig.json",
     "build:containers": "make build",
     "clean": "shx rm -rf dist",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "drop": "yarn redis:drop && yarn typeorm:drop && yarn typeorm:drop:issuer",
+    "redis:drop": "redis-cli flushall",
+    "deploy:heroku": "make -B build deploy-heroku",
     "prettier": "prettier --write --config-precedence file-override './src/**/*'",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "lint": "eslint \"src/**/*{.ts,.tsx}\" \"test/**/*{.ts,.tsx}\" --no-error-on-unmatched-pattern",
     "lint-fix": "eslint \"src/**/*{.ts,.tsx}\" \"test/**/*{.ts,.tsx}\" --fix --no-error-on-unmatched-pattern",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "migrate": "yarn typeorm:run:issuer && yarn typeorm:run",
+    "migrate:docker": "yarn typeorm:run:issuer && node_modules/typeorm/cli.js --config dist/js/ormconfig.js migration:run",
+    "typeorm": "ts-node -r tsconfig-paths/register node_modules/typeorm/cli.js --config ormconfig-dev.ts",
+    "typeorm:migrate": "yarn typeorm migration:generate -- -n",
+    "typeorm:run": "yarn typeorm migration:run",
+    "typeorm:run:issuer": "node_modules/typeorm/cli.js migration:run --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+    "typeorm:drop": "yarn typeorm schema:drop",
+    "typeorm:drop:issuer": "node_modules/typeorm/cli.js schema:drop --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+    "typeorm:dropAndMigrate": "yarn typeorm:drop && yarn typeorm:run",
+    "test:mocha": "mocha -r ts-node/register 'src/**/*.spec.ts' --timeout 60000 --exit",
+    "test:e2e:mocha": "mocha -r ts-node/register test/*.e2e-spec.ts --timeout 60000 --exit",
+    "test:e2e": "concurrently --success first --kill-others -n eth,test \"yarn run:ganache\" \"wait-on tcp:8545 && yarn typeorm:drop && yarn migrate && yarn test:e2e:mocha\"",
+    "test:e2e:single": "sh ./scripts/test-e2e-single.sh",
+    "run:ganache": "ganache-cli -q -m 'stone eyebrow bronze glide upon subject camera cushion rice outside swing casual' -l 8000000 -e 1000000 -a 20"
   },
   "bin": {
     "drec-backend": "./bin/drec-api"
   },
   "dependencies": {
+    "@energyweb/energy-api-influxdb": "0.7.1",
+    "@energyweb/issuer": "3.2.1-alpha.1623138698.0",
+    "@energyweb/issuer-api": "0.2.1-alpha.1623138698.0",
+    "@energyweb/origin-backend": "10.0.2-alpha.1623138698.0",
+    "@energyweb/origin-backend-core": "8.0.2-alpha.1623138698.0",
+    "@energyweb/origin-backend-utils": "1.5.2-alpha.1623138698.0",
+    "@energyweb/utils-general": "11.0.3-alpha.1623138698.0",
```

Comment:
Could you also use the latest releases with the ID 1625573089?
Thanks, there are some good improvements in there.